ProgramParsing/CONSTANTS/Constants.py

```diff
@@ -0,0 +1,9 @@
+"""
+Constants.py is a library built to receive information on Major Requirements
+
```

Comment:
Instead of CONSTANT let's actually call it ENV_VARIABLES - cuz we will be changing this in the future right? 

Also rename this file to something that is not constant.py since it's already under the CONSTANT (will become ENV_VARIABLES) folder. Prob something like parse_years (or smth)